### PR TITLE
[travis] upload teresa cli to release page

### PR DESCRIPTION
**Pull request by yagonobre**
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa/317)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,32 @@
...
+  - stage: helm
+    before_script:
+    - ./.helm.sh download
+    script:
+    - ./.helm.sh lint
+    after_success:
+    - ./.helm.sh deploy
+  - stage: build docker image
+    script:
+    - ./.ci-docker.sh
```
Comment (aguerra) [inline .travis.yml]: Are these indentation changes ok?

File: Makefile [hunk 2]
```diff
@@ -131,3 +131,14 @@ update-chart: helm-lint
...
+###Build Client in all Platforms
+PLATFORMS := windows linux darwin
+os = $(word 1, $@)
+
+.PHONY: $(PLATFORMS)
+$(PLATFORMS):
+	GOOS=$(os) go build -ldflags "-X $(BUILD_HOME)/pkg/version.Version=$(BUILD_VERSION)" -o teresa-$(os) $(BUILD_HOME)/cmd/client
+
+.PHONY: build-all-client
+build-all-client: windows linux darwin
```
Comment (aguerra) [inline Makefile]: I GUESS these `PHONY` targets are not necessary, but Makefiles are black magic

Comment (yagonobre) [reply Makefile]: Yes, probably this work without the `PHONE`, I will try :)

File: .travis.yml (hunk 1 above)
Comment (yagonobre) [reply .travis.yml]: My bad 😅

**Comment by yagonobre** [comment]
Closing this for now

**Pull request closed without merging**